<launch>
  <include file="$(find turtlebot_actions)/launch/server_turtlebot_move.launch"/>
  <node pkg="turtlesim" name="sim" type="turtlesim_node"/>
  <node pkg="assignment5" name="draw_triangle" type="assignment5_node" output="screen"/>
</launch>
